### DISCUSSION TITLE
Commands to start a anvil chain ?

### DISCUSSION THREAD

**Discussion by deepakA18** [Cyfrin/foundry-full-course-cu#333] [General]
I'm facing difficulty in deploying the contract on anvil:

command Used:-
''forge create SimpleStorage --rpc-url http://127.0.0.1:8545 --interactive"

Error:- 
No files changed, compilation skipped
Error: 
error sending request for url (http://127.0.0.1:8545/): error trying to connect: tcp connect error: Connection refused (os error 111)

Context:
- Error #0: error trying to connect: tcp connect error: Connection refused (os error 111)
- Error #1: tcp connect error: Connection refused (os error 111)
- Error #2: Connection refused (os error 111)

**Comment by shikhar229169** [comment]
Ensure that you have a running anvil chain in another terminal.
You can start an anvil chain by the command:
```
anvil
```

**Reply by cromewar** [reply]
Does this make sense for you @deepakA18 ?
